test/integration_test.js

```diff
@@ -22,7 +22,11 @@ describe("Integration tests", () => {
   var sandbox, server, tasks;
   const MAX_ATTEMPTS = 50;
 
-  function startServer(env={}) {
+  function startServer(env) {
+    // Add the provided environment variables to the child process environment.
+    // Keeping parent's environment is needed so that pserve's executable 
+    // can be found (with PATH) if KINTO_PSERVE_EXECUTABLE env variable was not provided.
```

Comment:
I'm reluctant landing this change as we lose control on what executable is actually used (imagine a globally installed `pserve` in a different version…)

I'd rather keep using an explicitly passed executable path, with proper documentation added (which is coming along #130 by @ametaireau).

Thoughts?